src/python/py_generate_pipeline.cpp

```diff
@@ -422,6 +424,7 @@ PYBIND11_MODULE(py_generate_pipeline, m) {
             ScopedVar env_manager(utils::ov_tokenizers_module_path());
             return std::make_unique<LLMPipeline>(model_path, tokenizer, device, utils::properties_to_any_map(config));
         }),
+        py::call_guard<py::scoped_ostream_redirect, py::scoped_estream_redirect>(),  // Respect std::cout flushes from constructor.
```

Comment:
does it need to be ported on master ?

Reply:
It was added to make constructor prints visible at the correct location. But the prints are commented out now and are going to be removed completely before the merge. That means it shouldn't be ported to master in that state.